src/compose-comquest-reducers.ts

```diff
@@ -2,11 +2,16 @@ import { AnyAction } from 'redux';
 import { StringIndexedObject } from './types';
 
 export function composeComquestReducers<
```

Comment:
Blimey

Reply:
It's not perfect, as it doesn't really give useful output if you go against the types, but it does prevent duplicate keys in reducers being composed. E.g. if you try to compose two request state reducers, it'll complain about the loading key:

<img width="454" alt="screen shot 2018-10-09 at 10 30 19" src="https://user-images.githubusercontent.com/5850625/46660121-42c8bd00-cbae-11e8-83d7-058d1a3ed8f9.png">